package ccc

// 这里 就会错误。 因为 这个 目录下 已经错在了 一个 目录下已经有了一个 main包 那么 这个目录下 就不允许 其他包
func main() {

}
